bigbluebutton-html5/imports/ui/components/plugins-engine/types.ts

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import * as PluginSdk from 'bigbluebutton-html-plugin-sdk';
+
+export interface PluginProvidedStateProps {
+    uuid: string
+}
+
+export interface PluginsEngineComponentProps {
+    containerRef: React.RefObject<HTMLDivElement>
+}
+
+export interface PluginLoaderComponentProps {
+    uuid: string
+    containerRef: React.RefObject<HTMLDivElement>
+    loadedPlugins: React.MutableRefObject<number>
+    setLastLoadedPlugin: React.Dispatch<React.SetStateAction<HTMLScriptElement | undefined>>
+    pluginConfigSettings: PluginConfigSetting
+}
+
+export interface PluginEngineProps {
+    onReady: Function
+}
+
+export interface PluginConfigSetting {
+    name: string
+    url: string
+}
+
+export interface UniquePluginConfigSetting {
+    name: string
+    url: string
+    uuid: string
+}
+
+export interface PluginObjects {
+    whiteboardToolbarItems: PluginSdk.WhiteboardToolbarItem[]
+}
```

Comment:
This type is used to hold the state provided by a specific plugin.

Said that, please use the name `PluginProvidedState` for this type.

Reply:
Hum, this seem to be the same case of the previous comment, to my view the objects are the same: 
```typescript
export interface ProvidedPlugins {
    whiteboardToolbarItems: PluginSdk.WhiteboardToolbarItem[];
}

export interface PluginObjects {
    whiteboardToolbarItems: PluginSdk.WhiteboardToolbarItem[]
}
```

Given that, I'd say we unify these into one to avoid code duplication and put a name suggested like `CurrentlyLoadedPlugins` or `PluginsProvidedCurrentState`

Reply:
+1 for reusing

replying about the `PluginsProvidedCurrentState` on the previous comment.